src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
The Lottery class should not extend ColorSupplier because Lottery is not a ColorSupplier.
Also, there are no methods that we can use or override in the Lottery class.